src/parser/jobs/brd/modules/OGCDDowntime.ts

```diff
@@ -6,9 +6,16 @@ import {CooldownDowntime} from 'parser/core/modules/CooldownDowntime'
 
 export default class OGCDDowntime extends CooldownDowntime {
 	trackedCds = [
-		{cooldowns: [ACTIONS.BARRAGE]},
-		{cooldowns: [ACTIONS.RAGING_STRIKES]},
-		{cooldowns: [ACTIONS.SIDEWINDER]},
+		{
+			cooldowns: [ACTIONS.BARRAGE],
+			firstUseOffset: 12000,
+		}, {
```

Comment:
Minor stylistic nit more than anything (so I won't block on it), but might be worth adjusting to have each opening `{` on their own line (which I see a lot of precedent for throughout the job modules I've been checking.

For example:

* [drg OGCDDowntime.js](https://github.com/xivanalysis/xivanalysis/blob/master/src/parser/jobs/drg/modules/OGCDDowntime.ts)
* [mnk Cooldowns.js](https://github.com/xivanalysis/xivanalysis/blob/master/src/parser/jobs/mnk/modules/Cooldowns.ts)

Other than that, perhaps set `defaultFirstUseOffset` to `12000` and just adjust the raging strikes cooldown?

Reply:
Gonna avoid overriding `defaultFirstUseOffset` in case we ever want to add other cooldowns (e.g. EA) in the future.

Reply:
Yeah, makes sense; I think it's good to go pending any other changes you'd like to make (I see Kelos making some suggestions for additional tracks, so I want to make sure before I approve)